From 6b5d4a218ce2e31b0730d45d9f6ddc0c14263e5e Mon Sep 17 00:00:00 2001
From: Yelena Krivosheev <yelena@marvell.com>
Date: Mon, 8 May 2017 13:24:01 +0300
Subject: [PATCH 1011/1345] fix: net: mvneta: use dev_info() in HW BM port
 init

commit  367091bd12b2442354dfab3475539422f756c1e3 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

netdev_info() cannot be used, net device didn't initialized
in this point.

Change-Id: I796de154bfd03bd9df9494f8bd1f20c5462a1860
Signed-off-by: Yelena Krivosheev <yelena@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/39235
Reviewed-by: Dmitri Epshtein <dima@marvell.com>
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Neta Zur Hershkovits <neta@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/39675
Reviewed-by: Victor Gu <xigu@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/ethernet/marvell/mvneta.c |   16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/drivers/net/ethernet/marvell/mvneta.c b/drivers/net/ethernet/marvell/mvneta.c
index 26443bc..10ffe03 100644
--- a/drivers/net/ethernet/marvell/mvneta.c
+++ b/drivers/net/ethernet/marvell/mvneta.c
@@ -1064,7 +1064,7 @@ static  int mvneta_bm_port_mbus_init(struct mvneta_port *pp)
 	err = mvneta_mbus_io_win_set(pp, pp->bm_priv->bppi_phys_addr, wsize,
 				     target, attr);
 	if (err < 0) {
-		netdev_info(pp->dev, "fail to configure mbus window to BM\n");
+		dev_info(&pdev->dev, "fail to configure mbus window to BM\n");
 		return err;
 	}
 	return 0;
@@ -1088,7 +1088,7 @@ static int mvneta_bm_port_init(struct platform_device *pdev,
 	}
 
 	if (of_property_read_u32(dn, "bm,pool-long", &long_pool_id)) {
-		netdev_info(pp->dev, "missing long pool id\n");
+		dev_info(&pdev->dev, "missing long pool id\n");
 		return -EINVAL;
 	}
 
@@ -1097,7 +1097,7 @@ static int mvneta_bm_port_init(struct platform_device *pdev,
 					   MVNETA_BM_LONG, pp->id,
 					   MVNETA_RX_PKT_SIZE(pp->dev->mtu));
 	if (!pp->pool_long) {
-		netdev_info(pp->dev, "fail to obtain long pool for port\n");
+		dev_info(&pdev->dev, "fail to obtain long pool for port\n");
 		return -ENOMEM;
 	}
 
@@ -1105,8 +1105,8 @@ static int mvneta_bm_port_init(struct platform_device *pdev,
 
 	mvneta_bm_pool_bufsize_set(pp, pp->pool_long->buf_size,
 				   pp->pool_long->id);
-	netdev_info(pp->dev, "create long pool %d for port %d, buffer size %d\n",
-		    pp->pool_long->id, pp->id, pp->pool_long->buf_size);
+	dev_info(&pdev->dev, "create long pool N%d, buffer size %d\n",
+		 pp->pool_long->id, pp->pool_long->buf_size);
 
 	/* If short pool id is not defined, assume using single pool */
 	if (of_property_read_u32(dn, "bm,pool-short", &short_pool_id))
@@ -1117,7 +1117,7 @@ static int mvneta_bm_port_init(struct platform_device *pdev,
 					    MVNETA_BM_SHORT, pp->id,
 					    MVNETA_BM_SHORT_PKT_SIZE);
 	if (!pp->pool_short) {
-		netdev_info(pp->dev, "fail to obtain short pool for port\n");
+		dev_info(&pdev->dev, "fail to obtain short pool for port\n");
 		mvneta_bm_pool_destroy(pp->bm_priv, pp->pool_long, 1 << pp->id);
 		return -ENOMEM;
 	}
@@ -1127,8 +1127,8 @@ static int mvneta_bm_port_init(struct platform_device *pdev,
 		mvneta_bm_pool_bufsize_set(pp, pp->pool_short->buf_size,
 					   pp->pool_short->id);
 	}
-	netdev_info(pp->dev, "create short pool %d for port %d, buffer size %d\n",
-		    pp->pool_short->id, pp->id, pp->pool_short->buf_size);
+	dev_info(&pdev->dev, "create short pool N%d, buffer size %d\n",
+		 pp->pool_short->id, pp->pool_short->buf_size);
 
 	return 0;
 }
-- 
1.7.9.5

